pkg/service/service_test.go

```diff
@@ -167,13 +167,11 @@ func TestGetServiceClassAndPlans(t *testing.T) {
 					Name:        "dev",
 					Description: "this is a example description 1",
 					DisplayName: "plan-name-1",
-					Required:    []string{"PLAN_DATABASE_URI", "PLAN_DATABASE_USERNAME", "PLAN_DATABASE_PASSWORD"},
```

Comment:
Please modify the tests according to the new changes :)

Reply:
I plan on doing that yes :+1:

Reply:
:+1:

Reply:
Fixed